README.md

```diff
@@ -75,7 +75,7 @@ A huge thanks to Carlos Hernandez who created the original Debian build configur
 
 A pre-built windows installer will be available soon. Until then it isn't too hard to install Jellyfin from Source.
 
-0. Install the dotnet core SDK 2.1 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/dotnet-core/2.1) and [install Git for Windows](https://gitforwindows.org/)
+0. Install the dotnet core SDK 2.2 from [Microsoft's Webpage](https://dotnet.microsoft.com/download/dotnet-core/2.2) and [install Git for Windows](https://gitforwindows.org/)
```

Comment:
lowercase 'w' on web page, 'webpage' should be two words.

Reply:
Separate PR / issue please.